src/service/controllers/messaging.js

```diff
@@ -33,17 +33,18 @@ exports.getCurrentSession = (req, res) => {
 		var user = conf.userByName(userName);
 
     if (!user.activeSession) {
-      res.status(403).end();
+      return res.status(403).end();
```

Comment:
This is a fix for a minor issue that runs the code below, even if the activeSession is not set.